index.js

```diff
@@ -0,0 +1,53 @@
+var map = document.querySelector(".contacts-map");
+var modalmap = document.querySelector(".modal-map");
```

Comment:
use camelCase for naming of vars consisted of two or more words

Reply:
Done